store/block.go

```diff
@@ -62,11 +67,24 @@ func (bs *blockStore) saveBlock(batch *leveldb.Batch, height uint32, block *bloc
 		regs[i].height = height
 		regs[i].offset = uint32(offset)
 
+		pubKey := trx.PublicKey()
+		if pubKey != nil {
+			if !bs.hasPublicKey(trx.Payload().Signer()) {
+				publicKeyKey := publicKeyKey(trx.Payload().Signer())
+				batch.Put(publicKeyKey, pubKey.Bytes())
+			} else {
+				// we have indexed this public key, se we can remove it
+				trx.SetPublicKey(nil)
```

Comment:
Here is a mistake: call `trx.StripPublicKey()` instead

Reply:
I should remove which one?
rx. or tx. ?